src/app/features/tech-record/create-batch-trl/components/batch-trl-details/batch-trl-details.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { DynamicFormsModule } from '@forms/dynamic-forms.module';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { SharedModule } from '@shared/shared.module';
+import { initialAppState, State } from '@store/.';
+import { BatchTrlDetailsComponent } from './batch-trl-details.component';
+
```

Comment:
These components definetly need some unit tests, I guess if we need this soon, we can forgo for now, but there are basically no tests on this